doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -51,6 +51,37 @@ GNU ARM Embedded
       C:\gnu_arm_embedded
 
 
+DesignWare ARC MetaWare Development Toolkit (MWDT)
+**************************************************
+
+#. You need to have ARC MWDT installed on your host.
```

Comment:
IMHO it worth pointing to a possibility for all interested parties to a possibility to get MWDT with 30 day evaluation license and also provide a guidance on how some may get a longer-lasting license becoming a part of "ARC Access Program". Something like that:
> It's possible to get evaluation license for "MetaWare Development Toolkit" here https://eval.synopsys.com/. It will last for 30 days.
>
> One may also apply for "ARC Access Program" (https://www.synopsys.com/designware-ip/processor-solutions/arc-access-program.html) and if approved to get a longer lasting license - that will be typically applicable to Synopsys partners or some active community members.

@ruuddw what do you think?

Reply:
I think a pointer to the MetaWare product page would be good to add (https://www.synopsys.com/dw/ipdir.php?ds=sw_metaware).
I don't see that mentioning the 30-day eval version, only MetaWare Lite (32kB code size limitation), so the eval pointer makes sense as well. Not sure about ARC Access - that would apply to a limited class of users only, who probably know about it anyway or can learn when asking for an eval license.